test/views/traderDetails/holderOfTransit/NameViewSpec.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.traderDetails.holderOfTransit
+
+import forms.NameFormProvider
+import models.NormalMode
+import play.api.data.Form
+import play.twirl.api.HtmlFormat
+import viewModels.InputSize
+import views.behaviours.InputTextViewBehaviours
+import views.html.traderDetails.holderOfTransit.NameView
+import org.scalacheck.{Arbitrary, Gen}
+
+class NameViewSpec extends InputTextViewBehaviours[String] {
+
+  override def form: Form[String] = new NameFormProvider()()
```

Comment:
This will then become `override def form: Form[String] = new NameFormProvider()(prefix)`